index.html

```diff
@@ -0,0 +1,198 @@
+          <li><a class="webinar" href="#">Webinar</a></li>
+          <li><img onclick="closeMenu()" class="showMenu close_btn" src="./imgs/close.png" alt=""></li>
+        </ul>
+      </nav>
+      <nav class="desktop_navbar">
+        <img  class="header_logo" src="./imgs/brain.png" alt="brain">
+        <ul class="navbar_lst">
+           <li><a href="index.html">Home</a></li>
+           <li><a href="#">Program</a></li>
+           <li><a href="about.html">About</a></li>
+           <li><a href="#">Sponsor</a></li>
+           <li><a href="#">News</a></li>
+         </ul>
+         <a class="webinar_btn" href="#">Webinar</a>
+      </nav>
+    </header>
+    <section class="containerOne">
+      <div class="imageDiv">
+        <h3>"Hello!Sharing World"</h3>
+        <h1>Creative commons <br> Summit<br>2015</h1>
+        <p>A joyful celebration believing in the value of openness and sharing, creating a postive  change with people from over 80 countries is taking place in october, in Korea.
+        </p>
+        <time datetime="2022-01-11T13:53">2015.10.15(THU) ~ 16(FRI)</time>
+        <h5>@ National Museum of Korea, Art Centre Nabi and more</h5>
+      </div>
+    </section>
+    <section class="programs">
+      <div class="center-container">
+        <h2>Main Program</h2>
+        <hr class="programLine" />
+      </div>
+      <div class="progContainer">
+        <div class="programCards">
+          <i class="fa fa-users fa-2x"></i>
+          <div><h3>Lecture</h3></div>
+          <p>
+            Many national governments around the world are showing themselves to
+            be inefficient
+          </p>
+        </div>
+        <div class="programCards">
+          <i class="fa fa-rocket fa-2x"></i>
+          <div><h3>Exhibition</h3></div>
+          <p>
+            Many national governments around the world are showing themselves to
+            be inefficient
+          </p>
+        </div>
+        <div class="programCards">
+          <i class="fa fa-comments-o fa-2x"></i>
+          <div><h3>Forum</h3></div>
+          <p>
+            Many national governments around the world are showing themselves to
+            be inefficient
+          </p>
+        </div>
+        <div class="programCards">
+          <i class="fa fa-microphone fa-2x"></i>
+          <div><h3>Workshop</h3></div>
+          <p>
+            Many national governments around the world are showing themselves to
+            be inefficient
+          </p>
+        </div>
+        <div class="programCards">
+          <i class="fa fa-graduation-cap fa-2x"></i>
+          <div><h3>Ignite</h3></div>
+          <div class="cardsColumn">
+            <p>
+              Many national governments around the world are showing themselves
+              to be inefficient
+            </p>
+          </div>
+        </div>
+      </div>
+      <div class="allPrograms">
+        <h3>JOIN THE EVENT</h3>
+        <hr />
+      </div>
+    </section>
```

Comment:
- [ ] AWESOME JOB 💎 so far but there are some things that need to be fixed and I am sure you can fix it in no time ⌚. In this section titles are creating a new line please kindly fix this issue 💯👍

<img width="1198" alt="Screen Shot 2021-12-25 at 5 24 33 PM" src="https://user-images.githubusercontent.com/69011963/147384784-b0c65124-5198-46e9-8b61-f10fbca39da7.png">